File: Язык программирования С++/Условный оператор/Простые сравнения/J.py
s = input() + '\n' + input() + '\n' + input()
b, r, g, y = 'black', 'red', 'green', 'yellow'
f1 = b + '\n' + b + '\n' + g
f2 =  b + '\n' + b + '\n' + 'GREEN'
f3 =  b + '\n' + y + '\n' + b
f4 = r + '\n' + b + '\n' + b
f5 = r + '\n' + y + '\n' + b
f6 = b + '\n' + 'YELLOW' + '\n'+ b
if s == f1:
  print(f2)
elif s == f2:
  print(f3)
elif s == f3:
    print(f4)
elif s == f4:
  print(f5)
elif s == f5:
  print(f1)
elif s == f6:
  print(f6)
else:
  print('error')

File: Язык программирования С++/Условный оператор/Простые сравнения/J.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
  string s1, s2, s3;
  cin >> s1 >> s2 >> s3;
  string s = s1 + "\n" + s2 + "\n" + s3;
  string b = "black", r = "red", g = "green", y =  "yellow";
  string f1 = b + "\n" + b + "\n" + g;
  string f2 =  b + "\n" + b + "\n" + "GREEN";
  string f3 =  b + "\n" + y + "\n" + b;
  string f4 = r + "\n" + b + "\n" + b;
  string f5 = r + "\n" + y + "\n" + b;
  string f6 = b + "\n" + "YELLOW" + "\n"+ b;
  if (s == f1) cout << f2;
  else if (s == f2) cout << f3;
  else if (s == f3) cout << f4;
  else if (s == f4) cout << f5;
  else if (s == f5) cout << f1;
  else if (s == f6) cout << f6;
  else cout << "error";
  return 0;
}